.github/workflows/build.yml

```diff
@@ -69,6 +69,7 @@ jobs:
     env:
       DOCKER_CLI_EXPERIMENTAL: "enabled"
       NO_TEST_PPC64LE: "true"
+      NO_TEST_S390X: "true"
```

Comment:
I don't think this is needed... we did that for ppc64le because some tests were failing only inside docker containers or something in that sense...

for s390x, it seems that we are already running the tests (line 67), so they are in fact passing already.